src/interaction/command/circle.ts

```diff
@@ -219,9 +204,14 @@ async function addCircle(bot: Bot, interaction: CommandInteraction) {
       type: "GUILD_TEXT",
       topic: desc,
       parent: circleCategory,
-      permissionOverwrites: permissions,
```

Comment:
why remove setting perms @ creation time, in favor of editing the perms after creating the channel?

Reply:
I wasn't able to change the permissions type to PermissionsOverwrite without doing it this way

Reply:
played around with it, fixed in [7ede779](https://github.com/acmutd/acm-bot/pull/80/commits/7ede77925fd5f72fca536597f4248fcbd848ded5)